content/java/concepts/strings/terms/subSequence/subSequence.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: '.subSequence()'
+Description: 'Returns a character sequence from a string.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+**`.subSquence()`** is a built-in method that returns a new character sequence that is a subsequence of a user-specified string.
+
+## Syntax
+
+```pseudo
+str.subSequence(int startIndex, int endIndex)
+```
+
+The `subSequence()` method will return a subsequence of the `str` string that starts with the character at the position `startIndex` and ends with the character at the position `endIndex -1`
```

Comment:
```suggestion
The `.subSequence()` method will return a subsequence of the `str` string from `startIndex` to`endIndex -1`
```

Reply:
Also, same comment as before: `.subSequence()` returns a substring, not a subsequence.